automation/prod-workflow-inc.sh

```diff
@@ -108,33 +125,18 @@ monitorSubmission() {
     name=$3
     submissionId=$4
 
-    ACCESS_TOKEN=$(
-        docker \
-            run \
-            --rm \
-            -v "${WORKING_DIR}:/app/populate" \
-            -w /app/populate broadinstitute/dsp-toolbox \
-            python get_bearer_token.py "${user}" "${JSON_CREDS}"
-    )
+    getAccessToken "$user"
 
-    submissionStatus=$(
-        curl \
-            -X GET \
-            --header 'Accept: application/json' \
-            --header "Authorization: Bearer ${ACCESS_TOKEN}" \
-            "https://api.firecloud.org/api/workspaces/$namespace/$name/submissions/$submissionId" \
-        | jq -r '.status'
-    )
-    workflowsStatus=$(
-        curl \
+    submissionDetails=$(curl \
             -X GET \
             --header 'Accept: application/json' \
             --header "Authorization: Bearer ${ACCESS_TOKEN}" \
-            "https://api.firecloud.org/api/workspaces/$namespace/$name/submissions/$submissionId" \
-        | jq -r '.workflows[] | .status'
-    )
+            "https://api.firecloud.org/api/workspaces/$namespace/$name/submissions/$submissionId")
```

Comment:
Not sure if this is already being done somewhere, but do we log the response of this curl? It would be helpful to log it in case we get an error apart from 401 from Rawls

Reply:
I originally had it explicitly `echo`ing these values out, but the bash script starts with `-x` which turns on debug logging for every line anyway. So it IS printed out, just not explicitly on every line in the script

Reply:
Ah I see. Thank you for explaining 👍